lib/objects.js

```diff
@@ -904,7 +915,8 @@ module.exports = {
 					16: 'Place me on flat surface',
 					17: 'Side brushes problem, reboot me',
 					18: 'Suction fan problem',
-					19: 'Unpowered charging station'
+					19: 'Unpowered charging station',
+					20: 'Remove Mop',
```

Comment:
müsste das nicht eher mop removed heissen? Ist doch ein Fehlerstatus

Reply:
Nein, nach dem säubern, möchte er dass der Mop entfernt und gesäubert wird.

Bin gerade zufällig in dem Error.